Bot_project/main.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream>
+#include <string>
+
+using namespace std; //I know using namespace std is bad practice, but i'm just testing stuff out.
+
+//a function that doesn't ask for a name
+//so I can call and return for a loop
+//(so the program stays awake).
+//Definitely not the best way but I'll change it later.
+
+int prompt(const string& input_name )
```

Comment:
input_name can be user_name for clarity...
input can be user_message or something along those lines...
no change in functionality, but it'll be easier to read for the next person :)